source/src/molecular-unfolding/cdk/utils/utils-role.ts

```diff
@@ -117,6 +117,16 @@ export class RoleUtil {
       ],
     }));
 
+    role.addToPolicy(new iam.PolicyStatement({
```

Comment:
should the bootstrapless synthesizer do it ?

Reply:
gcr pipeline can do it automatically. but global pipeline cannot do it, so add it here.

Reply:
Are all below buckets with `-gcr-` used in pipeline?